core-ui/src/components/Predefined/Create/LimitRanges/templates.js

```diff
@@ -1,6 +1,7 @@
 export function createLimitRangeTemplate({
   max = '1100Mi',
-  defaultVal = '511Mi',
+  min = '32Mi',
```

Comment:
Maybe we should modify tests to check if created limit range uses these defaults values? For now, we only check the name.

Reply:
I didn't check tests, I will look at it later.